tests/test_worktree_cleanup.sh

```
@@ -0,0 +1,379 @@
+
+if [ "$RC2" -eq 0 ]; then
+  pass "apply without escalation exits 0"
+else
+  fail "apply exit code $RC2, expected 0"
+  echo "$OUT2" >&2
+fi
+
+# Re-run dry-run and re-check state.
+set +e
+OUT3=$(PATH="$STUB_DIR:$PATH" bash "$HELPER" --no-color --dry-run 2>&1)
+RC3=$?
```

Shellcheck (0.11.0)

[warning] RC3 appears unused. Verify use (or export if used externally).

(SC2034)